### PR TITLE
Enable Structured Sparseity B Matrix

### PR DESCRIPTION
**Pull request by vin-huang**

### CODE REVIEW

File: clients/include/spmm/testing_spmm.hpp [hunk 1]
```diff
@@ -282,22 +282,29 @@ void testing_spmm(const Arguments& arg)
...
+                                     arg.b_type,
+                                     HIPSPARSE_ORDER_COL);
     hipsparselt_local_mat_descr matC(
         hipsparselt_matrix_type_dense, handle, M, N, ldc, arg.c_type, HIPSPARSE_ORDER_COL);
     hipsparselt_local_mat_descr matD(
         hipsparselt_matrix_type_dense, handle, M, N, ldc, arg.d_type, HIPSPARSE_ORDER_COL);
 
     hipsparseStatus_t eStatus
-        = expected_hipsparse_status_of_matrix_size(arg.a_type, A_row, A_col, lda);
+        = expected_hipsparse_status_of_matrix_size(arg.a_type, A_row, A_col, lda, true);
```
Comment (hcman2) [inline clients/include/spmm/testing_spmm.hpp]: Should this "true" be "!arg.sparse_b"?

File: clients/include/spmm/testing_spmm.hpp [hunk 2]
```diff
@@ -940,7 +966,7 @@ void testing_aux_plan_assign(const Arguments& arg)
...
```
Comment (hcman2) [inline clients/include/spmm/testing_spmm.hpp]: Should this "true" be "!arg.sparse_b"?

Comment (vin-huang) [reply clients/include/spmm/testing_spmm.hpp]: yes, it should be changed.

File: clients/include/spmm/testing_spmm.hpp (hunk 1 above)
Comment (vin-huang) [reply clients/include/spmm/testing_spmm.hpp]: yes, it should be changed.

**Review by hcman2 — APPROVED** [review]
LGTM

**Pull request merged**